CONTRIBUTING.md

```diff
@@ -210,32 +355,134 @@ public boolean pushedByWater() {
+progress will be lost if you do not;
+1. Identify the name(s) of the file(s) you want to import.
+   - A complete list of all possible file names can be found at
+   `./work/Minecraft/$MCVER/spigot/net/minecraft/server`. You might find
+   [MiniMappingViewer] very useful to find the file you need.
+1. Open the file at `./scripts/importmcdev.sh` and add the name of your file to
+the script. Skip to the 2nd last header and follow the instructions there;
+1. Re-patch the server `./paper patch`;
+1. Edit away!
+
+❗ This change is temporary! **DO NOT COMMIT CHANGES TO THIS FILE!**  
+Once you have made your changes to the new file, and rebuilt patches, you may
+undo your changes to `importmcdev.sh`.
+
+Any file modified in a patch file gets automatically imported, so you only need
+this temporarily to import it to create the first patch.
+
+To undo your changes to the file, type `git checkout scripts/importmcdev.sh`.
+
+### Where can I learn what this method/field does?
+
+For most cases, it is preferred if you use [yarn], as their license works with
+Paper's license. If you can't do that, [MiniMappingViewer] is always around to
+provide you with more of a general idea, *but* you cannot use the Mojang names
+for more than understanding the code.
+
+[yarn] is in general more thorough than Mojang's own mappings, as they include
+method arguments as well, whereas Mojang's do not. If you need local variables
+to understand the code, you might be more lucky with ModCoderPack.
+
+For more information on the Mojang name licensing issues, check this out:
+<https://cpw.github.io/MinecraftMappingData.html>
+
+### My commit doesn't need a build, what do I do?
+
+Well, quite simple: You add `[CI-SKIP]` to the start of your commit subject.
+
+This case most often applies to changes to files like `README.md`, this very
+file (`CONTRIBUTING.md`), the `LICENSE.md` file, and so forth.
+
+### Patching and building is *really* slow, what can I do?
+
+This only applies if you're running Windows 10. If you're running a prior
+Windows release, either update to Windows 10 or move to macOS/Linux/BSD.
+
+To speed up the programs used, you have to get WSL 2. This is available in
+Windows 10 v2004, build 19041 or higher. Check your version by running `winver`
+in the run window (Windows key + R). If you're out of date, update your system
+with the Windows Update Assistant.
+
+To get WSL 2, follow the information at this link:
+<https://docs.microsoft.com/en-us/windows/wsl/install-win10>
+
+You will most likely get most out of using the Ubuntu apps. Once it's set up,
+install the required toolswith `sudo apt-get update && sudo apt-get install
+git patch default-jdk maven -y`. You can now clone the repository and do
+everything like usual.
+
+❗ Do not use the `/mnt/` directory in WSL! Instead, mount the WSL directories
+in Windows like described here:
+<https://www.howtogeek.com/426749/how-to-access-your-linux-wsl-files-in-windows-10/>
```

Comment:
```suggestion
> ❗ Do not use the `/mnt/` directory in WSL! Instead, mount the WSL directories
in Windows like described here:
<https://www.howtogeek.com/426749/how-to-access-your-linux-wsl-files-in-windows-10/>
```
Placing noteworthy messages in blockquotes will make them more noticeable than just an emoji at the start of the sentence